eclipse-mosquitto/content.md

```diff
@@ -0,0 +1,42 @@
+# What is Eclipse Mosquitto?
+
+Eclipse Mosquitto is an open source implementation of a server for version 3.1 and 3.1.1 of the MQTT protocol. Main homepage: http://mosquitto.org/
+
+%%LOGO%%
+
+# How to use this image
+
+##Mount Points
+
+Three mount points have been created in the image to be used for configuration, persistent storage and logs.
```

Comment:
Calling these "mount points" might be a little confusing for users -- in my mind, "mount points" would imply that these are defined as volumes, but they're not.

Perhaps just calling them "directories" or "locations" would be better?